src/main/kotlin/pet/project/app/dto/book/RequestCreateBookDto.kt

```diff
@@ -0,0 +1,9 @@
+package pet.project.app.dto.book
+
+data class RequestCreateBookDto(
```

Comment:
```suggestion
data class CreateBookRequest(
```
Dto doesn't have any semantic value here